nss-day-cohort-29/reactive-nutshell-springwynne
# Ac setting up tasks

`# Description

When user clicks on Task link in Navigation Bar, they are "re-directed" to a page that displays all tasks. 
When user clicks on "Add Task" button, user is "re-directed" to a page that says "It's working"

## Type of change

- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions for Change Made

1. Git fetch --all
1. Git checkout ac-setting-up-tasks
1. Click on Task in nav bar, make sure url is now "localhost:3000/tasks"
1. Make sure localhost:3000/tasks displays a button that says "Add Task" and displays all of users tasks (currently hard-coded)
1. In the list of displayed tasks, make sure each task has a title and "completion date"
1. Click "Add Task" button 
1. When button is clicked, make sure user is "re-directed" to "localhost:3000/tasks/new"
1. When at localhost:3000/tasks/new confirm that "It's working" is displayed on page 

# Checklist:

- [ X ] My code follows the style guidelines of this project
- [ X ] I have performed a self-review of my own code
- [ X ] My changes generate no new warnings
- [ X ] I have added test instructions that prove my fix is effective or that my feature works`
